update_kb_redis.py

```diff
@@ -509,12 +519,26 @@ def update_kb_redis():
         config_h = parse_config_h(keyboard)
         rules_mk = parse_rules_mk(keyboard)
 
+        usb_entry = {'keyboard': keyboard}
         for key in ('VENDOR_ID', 'PRODUCT_ID', 'DEVICE_VER', 'MANUFACTURER', 'DESCRIPTION'):
             if key in config_h:
                 if key in ('VENDOR_ID', 'PRODUCT_ID', 'DEVICE_VER'):
-                    config_h[key].replace('0x', '')
-                    config_h[key] = config_h[key].upper()
+                    config_h[key] = config_h[key].upper().replace('0X', '')
```

Comment:
save a line, just replace it with 0x instead of empty?

Reply:
I also want to catch the case where neither '0x' or '0X' appear.